src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+$color-blue: #00acdc;
+$color-dark: #060b35;
+$color-gray: #616070;
+$color-white: #fff;
+$color-card-border: #f3f3f3;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border: 1px solid $color-card-border;
+  border-radius: 5px;
+  background-color: $color-white;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 40px;
+  }
+
+  &__description {
+    margin: 0 16px;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-dark;
+  }
+
+  &__code {
+    margin: 0 16px 16px;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $color-gray;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 16px 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 16px 16px;
+  }
+
+  &__price {
+    align-items: flex-end;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-gray;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    text-align: right;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-dark;
+  }
```

Comment:
format your code and fix all cases if present
![image](https://user-images.githubusercontent.com/70009205/236502559-f4f1b234-3250-4590-bc4b-96c6cada8314.png)